ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-ref.yaml

```diff
@@ -7,9 +7,8 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
-    - name: OS_CLOUD
-      documentation: Name of cloud to use from ${CLUSTER_PROFILE_DIR}/clouds.yaml file
     - name: OPENSTACK_EXTERNAL_NETWORK
+      default: "external"
```

Comment:
added default value. Most of the time is is "external" but it can still be changed anywhere by any chain or workflow so users can use something else if needed.